bin/build-cloud

```diff
@@ -0,0 +1,35 @@
+ #! /usr/bin/env bash
+
+ # DOC: Build civiform/cloud.
+
+IMAGE="cloud"
```

Comment:
I feel like "cloud" isn't the best name for this.  It's generic, and doesn't really describe what the image does (i.e. `docker run cloud` means almost nothing)